packages/base/card-api.gts

```diff
@@ -1322,8 +1322,8 @@ class LinksToMany<FieldT extends CardDefConstructor>
         }
         let resourceId = new URL(
           value.links.self,
-          !Array.isArray(doc.data) && 'id' in doc.data && doc.data.id
-            ? doc.data.id
+          !Array.isArray(value.data) && value.data?.id
+            ? value?.data?.id
```

Comment:
this should just be value?.data?.id. do _not_ consider `links.self` at all.

Reply:
here we shudnt be relative to anything at all

Reply:
Yes. This is the key ingredient